content/projects/knowtox.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "KnowTox"
+date: 2020-05-20T16:27:08+02:00
+
+weight: 10
+nav: false
+
+people:
+- key: andrea-lilian.morger
+- key: andrea.volkamer
+
+collaborators:
+- name: Miriam Mathea
+  link: https://www.basf.com/
+- name: Janosch Achenbach
+  link: https://www.basf.com/
+- name: Robert Landsiedel
+  link: https://www.basf.com/
+- name: Roland Buesen
+  link: https://www.basf.com/
+- name: Antje Wolf
+  link: https://www.basf.com/
+- name: Klaus-Juergen Schleifer
+  link: https://www.basf.com/
+
+
+publications:
+- morger_2020_knowtox
+
+external_resources:
+- name: knowtox_manuscript_SI
+  link: https://github.com/volkamerlab/knowtox_manuscript_SI
+  icon: github
+  more: SI notebook - build & evaluate conformal predictor and apply to external data
+
+funding:
+- name: Bundesministerium für Bildung und Forschung, grant ID 031A262C
+- name: HaVo-Stiftung
+- name: We thank BASF for supporting this project by providing financial resources
+
+summary: |
+  KnowTox is a novel pipeline that combines three different _in silico_ toxicology approaches to allow for confident prediction of potentially toxic effects of novel chemical entities, i.e. machine learning models, alerts for toxic substructures and computational support for read-across.
+---
+
+Computational tools for toxicity prediction are promising in the process of reducing, refining and replacing animal testing. To assess the toxicity of novel chemical entities, regulatory agencies require in vivo testing for several toxic endpoints. In 2010, roughly 2.9 million laboratory animals have been deployed in Germany, with an increase of 6% since 2008. Thus, the establishment of alternative methods, and with it the reduction of animal testing, is of utmost importance.
```

Comment:
```suggestion
Computational tools for toxicity prediction are promising in the process of reducing, refining and replacing animal testing. To assess the toxicity of novel chemical entities, regulatory agencies require in vivo testing for several toxic endpoints. In 2017, roughly 2.19 million laboratory animals have been deployed in Germany for regulatory testing purposes (~11 % of those for testing industrial chemicals). Thus, the establishment of alternative methods, and with it the reduction of animal testing, is of utmost importance.
```
add respective link: https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1581689520921&uri=CELEX:52020DC0016

Reply:
* I would change `11 %` to `11%`
* `of utmost importance` is used again in the next paragraph, i.e. `When applying machine learning models, applicability and reliability of predictions for new chemicals are of utmost importance.` Could we change one of them with e.g. `much-needed` or `of extreme importance`?

* If I understand the report correctly, the new numbers are for the European Union, so we should exchange `in Germany` by `in the European Union` (see Eu Report animal use, p. 13)  - do you agree @AndreaVolkamer ?